ocaml/xapi/xapi_cluster_host_helpers.ml

```diff
@@ -56,6 +56,7 @@ let get_operation_error ~__context ~self ~op =
   current_error
 
 let assert_operation_valid ~__context ~self ~op =
+  Helpers.assert_rolling_upgrade_not_in_progress ~__context;
```

Comment:
We have to allow shutdown/reboot, i.e. disable/enable during a rolling pool upgrade, I think that is why the commented out code below had a list of what to allow.

Reply:
So we should allow all cluster_host operations during RPU and just prohibit cluster operations?

Reply:
Yes, we want to prevent membership changes, but host lifecycle changes should be fine.